builtin/settingtypes.txt

```diff
@@ -71,7 +71,7 @@ fast_move (Fast movement) bool false
 noclip (Noclip) bool false
 
 #    Smooths camera when moving and looking around.
-#    Useful for recording videos.
+#    Useful for recording videos. Aka mouse or look smoothing.
```

Comment:
Okay a mention of 'mouse smoothing' is okay by me, the text still needs improving.